src/config/enhance-octokit.ts

```diff
@@ -29,11 +29,11 @@ const instrumentRequests = (octokit: GitHubAPI) => {
 			throw new RateLimitingError(rateLimitReset);
 		}
 
-		if (error.status === 403) {
-			// delaying for an hour
-			logger.warn({ err: error }, "GitHub returned 403");
-			throw new RateLimitingError(new Date().getTime() / 1000 + 60 * 60);
+		if (error.status === 403 && error.stack?.includes("has an IP allow list enabled")) {
+			logger.warn({ err: error }, "blocked by GitHub allowlist!");
+			statsd.increment(metricError.blockedByGitHubAllowlist);
 		}
```

Comment:
Now, we're checking the stack trace for a special error message from GitHub and emit a metric to make IP allowlist errors more prominent (I'll later add a low-prio alert on that metric)

Reply:
Tested the metric and new log event in staging.